src/guides/v2.3/rest/tutorials/orders/order-add-items.md

```diff
@@ -33,9 +33,9 @@ The following example adds an orange medium-sized Radiant women's t-shirt (`sku`
 
 **Headers:**
 
-`Content-Type` `application/json`
+`Content-Type`: `application/json`
 
-`Authorization` `Bearer <customer token>`
+`Authorization`: `Bearer` `<customer` `token>`
```

Comment:
No need to split <customer token>. Please revert them.

Reply:
Hello @dobooth.
Thank you for your remark.
I fixed it.
Please check.
Thank you in advance.